### PR TITLE
Logstash package

### PR DESCRIPTION
**Pull request by sayden**
## What does this PR do?

Add Logstash integration

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Related issues

- Relates https://github.com/elastic/kibana/issues/105042

## Screenshots

image

image

image

### CODE REVIEW

File: packages/logstash/data_stream/log/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+type: logs
+title: logstash log logs
```
Comment (mtojek) [inline packages/logstash/data_stream/log/manifest.yml:2]: nit: this sounds a bit weird :)

File: packages/logstash/data_stream/log/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        title: Paths
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - /var/log/logstash/logstash-plain*.log
+          - /var/log/logstash/logstash-json*.log
+    template_path: log.yml.hbs
+    title: Logstash logs
+    description: Collect Logstash logs using the log input
```
Comment (mtojek) [inline packages/logstash/data_stream/log/manifest.yml:18]: nit: maybe "Collect Logstash logs from standard files?"

File: packages/logstash/data_stream/node/fields/ecs.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  external: ecs
+- name: service.id
+  external: ecs
+- name: service.type
+  external: ecs
+- name: service.version
+  external: ecs
+- name: service.address
+  type: keyword
+  description: Address where data about this service was collected from.
```
Comment (mtojek) [inline packages/logstash/data_stream/node/fields/ecs.yml:11]: FYI I will be available in 1.12: https://github.com/elastic/ecs/issues/1534

File: packages/logstash/data_stream/slowlog/sample_event.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mtojek) [inline packages/logstash/data_stream/slowlog/sample_event.json:1]: Is is on purpose empty?

File: packages/logstash/data_stream/log/manifest.yml (hunk 2 above)
Comment (sayden) [reply packages/logstash/data_stream/log/manifest.yml]: Good suggestion, thank you!

File: packages/logstash/data_stream/slowlog/sample_event.json (hunk 4 above)
Comment (sayden) [reply packages/logstash/data_stream/slowlog/sample_event.json]: Nope, thanks! I ran into some issues and I'll probably update this in some follow-up.

File: packages/logstash/data_stream/log/manifest.yml (hunk 1 above)
Comment (sayden) [reply packages/logstash/data_stream/log/manifest.yml]: Sooo many titles everywhere. I fixed it, thank you for spotting

File: packages/logstash/_dev/build/build.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.11
```
Comment (mtojek) [inline packages/logstash/_dev/build/build.yml:3]: nit: I guess you can update it to 1.12. It should be available.

**Review by mtojek — APPROVED** [review]

**Comment by sayden** [comment]
ECS 1.12 will be updated in a follow up, once it is released in Github. Merging is possible

**Pull request merged**